h2o-py/h2o/explanation/_explain.py

```diff
@@ -1253,7 +1258,8 @@ def ice_plot(
     :param max_levels: maximum number of factor levels to show
     :param figsize: figure size; passed directly to matplotlib
     :param colormap: colormap name
-    :param save_plot_path: a path to save the plot via using mathplotlib function savefig  
+    :param save_plot_path: a path to save the plot via using mathplotlib function savefig
+    :param centered: whether to normalize y-axis values or not
```

Comment:
QQ: Excuse me for lack of knowledge, but just to clarify, is this really called a **normalization**? I'm referring to a definition of _normalization_ that states that normalization is a technique, which scales values to a common scale, which is not really the case here. 🙂  Thanks!

Reply:
it is a formulation from the requirement, will reformulate to be more exact 👍

Reply:
[8438350](https://github.com/h2oai/h2o-3/pull/6059/commits/8438350b8fb91a2bddbdc12272173b5877de28b3)